README.md

```diff
@@ -1,227 +1,68 @@
-* unsub, unsubscribe Unsubscribe this MQTT client from a list of topics
-* dis, disconnect Disconnect this MQTT client
-* switch Switch the current context
-* ls, list List all connected clients of this mqtt-cli session with their respective identifiers
-* cls, clear Clear the screen
-* exit Exit the current context
+<img align="center" src="./img/shell.gif"/>
 
 ## Test
 
-The test command runs various tests against the specified broker to find out its features and limitations.
-By default the test command will use MQTT 3 clients to test the broker first
-and will afterwards check the connect restrictions returned by a connect of a MQTT 5 client.
-You can alter this behavior by specifying different options when using the command.
-
-Test the public HiveMQ broker:
-
-``` 
-$ mqtt test -h broker.hivemq.com
-MQTT 3: OK
-        - Maximum topic length: 65535 bytes
-        - QoS 0: Received 10/10 publishes in 25,74ms
-        - QoS 1: Received 10/10 publishes in 26,27ms
-        - QoS 2: Received 10/10 publishes in 70,01ms
-        - Retain: OK
-        - Wildcard subscriptions: OK
-        - Shared subscriptions: OK
-        - Payload size: >= 100000 bytes
-        - Maximum client id length: 65535 bytes
-        - Unsupported Ascii Chars: ALL SUPPORTED
-MQTT 5: OK
-        - Connect restrictions: 
-                > Retain: OK
-                > Wildcard subscriptions: OK
-                > Shared subscriptions: OK
-                > Subscription identifiers: OK
-                > Maximum QoS: 2
-                > Receive maximum: 10
-                > Maximum packet size: 268435460 bytes
-                > Topic alias maximum: 5
-                > Session expiry interval: Client-based
-                > Server keep alive: Client-based
-
-```
-
-## HiveMQ
-
-The HiveMQ command line argument offers various HiveMQ specific commands.
-
-Show all available commands:
-
-``$ mqtt hivemq``
-
-The export command of the HiveMQ command line offers a set of commands to export various resources from a HiveMQ API
-endpoint.
-
-``$ mqtt hivemq export``
-
-Export client details from a HiveMQ node via the export clients command.
-
-``$ mqtt hivemq export clients``
-
-Please refer to the detailed documentation [MQTT CLI Documentation](https://hivemq.github.io/mqtt-cli)
-for more examples and complete command descriptions.
-
-## Versioning
-
-[Semantic Versioning](https://semver.org/) is used.
+- Run tests against a broker to find out its features and limitations
+- Quick start: `mqtt test -h your-mqtt-broker.com`
+- [Further documentation](https://hivemq.github.io/mqtt-cli/docs/test/)
 
-## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+<img align="center" src="./img/test.gif"/>
 
-## License
 
-See [LICENSE](LICENSE)
+## How to contribute
+- If you want to request a feature or report a bug, please [create a GitHub Issue using one of the provided templates](https://github.com/hivemq/mqtt-cli/issues/new/choose)
```

Comment:
issue

Reply:
GitHub Issue is usually written in upper case.